core/src/cg/group4/game_logic/StandUp.java

```diff
@@ -31,11 +32,6 @@
     protected static StandUp cInstance;
 
     /**
-     * Background music to be played.
-     */
-    protected Music cBackgroundMusic;
```

Comment:
Good! Now also the sensor reader and acceleration status.
Wouldn't it be better to have a class that keeps track of such object, a so called container class?